app/_components/systems/Form.tsx

```diff
@@ -0,0 +1,334 @@
+              </FormControl>
+            )}
+          />
+        </GridItem>
+
+        <GridItem w="100%">
+          <AllegiancesField control={control} />
+        </GridItem>
+
+        <GridItem w="100%">
+          <GovernmentsField control={control} />
+        </GridItem>
+
+        <GridItem w="100%">
+          <EconomiesField control={control} />
+        </GridItem>
+
+        <GridItem w="100%" colSpan={{ base: 1, md: 2, lg: 3 }}>
+          <FormControl
+            isInvalid={!!(errors.minorFaction && errors.minorFaction.message)}
+          >
+            <FormLabel>Minor Faction</FormLabel>
+            <Input
+              variant="outline"
+              placeholder="Search for a minor faction..."
+              borderColor={selectColor(isDark, 'border')}
+              _hover={{
+                borderColor: selectColor(isDark, 'border'),
+              }}
+              {...register('minorFaction', {
+                required: true,
+                pattern: /^[\w'-]+(?:\s[\w'-]+)*$/,
+                maxLength: 40,
+              })}
+            />
+            <FormErrorMessage>
+              {errors.minorFaction && errors.minorFaction.message}
+            </FormErrorMessage>
+          </FormControl>
+        </GridItem>
+
+        <GridItem w="100%">
+          <Controller
+            name="presenceType"
+            control={control}
+            render={({
+              field: { onChange, onBlur, value, name, ref },
+              fieldState: { error },
+            }) => (
+              <FormControl isInvalid={!!error} id="presenceType">
+                <FormLabel>Presence Type</FormLabel>
+                <Select<SelectGroup, true, GroupBase<SelectGroup>>
+                  id="presenceType-field"
+                  instanceId="presenceType-field"
+                  name={name}
+                  ref={ref}
+                  onBlur={onBlur}
+                  value={value}
+                  options={[
+                    { label: 'Presence', value: 'presence' },
+                    { label: 'Controlling', value: 'controlling' },
+                  ]}
+                  chakraStyles={chakraSelectStyles}
+                />
+                <FormErrorMessage>{error && error.message}</FormErrorMessage>
+              </FormControl>
+            )}
+          />
+        </GridItem>
+
+        <GridItem w="100%">
+          <RequiresPermitField control={control} />
+        </GridItem>
+
+        <GridItem w="100%">
+          <Controller
+            name="stationFilter"
+            control={control}
+            render={({
+              field: { onChange, onBlur, value, name, ref },
+              fieldState: { error },
+            }) => (
+              <FormControl isInvalid={!!error} id="stationFilter">
+                <FormLabel>Station Filter</FormLabel>
+                <Select<SelectGroup, true, GroupBase<SelectGroup>>
```

Comment:
This is subjective but I think radio buttons could work better here so the user can clearly see their options without a click. Also, what if we want orbital and planetary? Perhaps it should be checkboxes instead.

Reply:
I agree completely, but am going to leave this for now as it moves into the realms of allow multiselect, which is beyond EDDB feature parity. We can revisit add this later if support for it is added on the API